WISE-Community/WISE-Client
# chore(Classroom Monitor): Convert Toolbar

## Changes

Converted Classroom Monitor Toolbar to Angular.

## Test

1. Open the Classroom Monitor for a run
2. Make sure the "Main Menu" button near the top left slides out the menu
3. Go to the "Grade By Step" view and click on a step. The step select drop down should show up in the toolbar along with the previous and next buttons.
4. Go to the "Grade By Team" view and click on a team. The team select drop down should show up in the toolbar along with the previous and next buttons.
5. Make sure the period drop down near the top right properly filters workgroups that are in the selected period

Closes #807
